when i run 
#! /usr/bin/env python3
import rclpy
from rclpy.node import Node
import subprocess

class RobotBringup(Node):
    def __init__(self):
        super().__init__('robot_bringup')
        # SSH command
        self.robot_hostname = '192.168.1.21'  
        self.robot_user = 'turtlebot' 
        self.model = 'waffle'   

        self.bring_up_robot()

    def bring_up_robot(self):

        command = (
                f"ssh {self.robot_user}@{self.robot_hostname} 'echo Hello world'"
                "'export TURTLEBOT3_MODEL=waffle ; echo $TURTLEBOT3_MODEL'"
                # f"source /opt/ros/foxy/setup.bash &&"
                # f"ros2 launch turtlebot3_bringup robot.launch.py"
        )
        try:
            subprocess.run(command, shell=True, check=True)
            self.get_logger().info("Robot bringup initiated successfully.")
        except subprocess.CalledProcessError as e:
            self.get_logger().error(f"Failed to start robot: {str(e)}")

def main(args=None):
    rclpy.init(args=args)
    robot_bringup = RobotBringup()
    rclpy.spin(robot_bringup)
    robot_bringup.destroy_node()
    rclpy.shutdown()

if __name__ == '__main__':
    main()


i get 

ros2 run navigation_turtlebot robot_bringup.py 
turtlebot@192.168.1.21's password: 
Hello worldexport TURTLEBOT3_MODEL=waffle

[INFO] [1713772311.205079879] [robot_bringup]: Robot bringup initiated successfully.
